Tech Journey/My-#100DayOfCode-Journey-So-Far.md

```diff
@@ -0,0 +1,56 @@
+## A bit about me, Who am I?
+Hey, my name is **Ameen**, I am 15 years old and currently in high school. I started learning to code when I was in second grade in school but never took it seriously until I was 11... That time I started learning **Unity** ( A Game Engine ) and **C#** but due to my PC's specs I couldn't continue on that path so instead of giving up  I started learning **Python**, learned **PyGame** and a few more libs and built something basic with each of em.
+But I didn't enjoy it as much and started to lose my interest in python. So the hunt to find something fun had begun again. After exploring for a while, I found "**Web Development**", when I started learning web development it just clicked with me didn't take me much time to master HTML but then school came in the way . 2 years later I again continued the web dev path with the #100DaysOfCode challenge and am having fun ever since.
```

Comment:
It should be in a new paragraph, shift it to line number 4.  

"But I didn't enjoy it as much and started to lose my interest in python. So the hunt to find something fun had begun again." is too long. You can also say "After losing interest in python, I started my hunt again"

"After exploring for a while, I found "**Web Development**" to "I found **Web Development** after exploring" - same reason, keep your sentences short while conveying the same meaning.

"when I started learning web development it just clicked with me didn't take me much time to master HTML but then school came in the way " -- too long, make it short. I'll let you do this one in a similar way as above. Let me know if you need any help